flopy/utils/check.py

```diff
@@ -772,6 +772,37 @@ def get_neighbors(a):
     return neighbors.reshape(6, nk, ni, nj)
 
 
+def get_neighbors_u(a, disu):
+    """
+    Returns the n neighboring values for each value in a.
+
+    Parameters
+    ----------
+    a : 1-D numpy array of values for all nodes in the model
+        Model array in node order.
+    disu : flopy mfusg disu package
+
+
+    Returns
+    -------
+    neighbors : jagged list for each node in a, containing numpy arrays of
+        neighbouring values from a.
+    """
+
+    ja = disu.ja.array - 1
+    iac_sum = np.cumsum(disu.iac.array)
+    ja_slices = np.asarray(
+        [
+            np.s_[iac_sum[i - 1] + 1 : x] if i > 0 else np.s_[1:x]
+            for i, x in enumerate(iac_sum)
+        ]
+    )  # note: this removes the diagonal - neighbors only
+    neighbors = [ja[sl] for sl in ja_slices]
+    a_neighbors = [a[n] for n in neighbors]
+
+    return a_neighbors
+
+
```

Comment:
If "get_neighbors" is generic code then I think we should move it to the different types of model grid.  Or maybe for mf2005/mfusg/mfnwt it just goes in the dis and disu packages?